transport/src/main/java/io/netty/channel/ChannelHandlerMask.java

```diff
@@ -61,15 +61,20 @@ final class ChannelHandlerMask {
     static final int MASK_ONLY_OUTBOUND =  MASK_BIND | MASK_CONNECT | MASK_DISCONNECT |
             MASK_CLOSE | MASK_DEREGISTER | MASK_READ | MASK_WRITE | MASK_FLUSH;
     private static final int MASK_ALL_OUTBOUND = MASK_EXCEPTION_CAUGHT | MASK_ONLY_OUTBOUND;
+    private static final int MASK_BITS = 17;
```

Comment:
Oh no!! this is what would make the branchless approach most costly since division by 17 could be improved but cannot be as fast as a power of 2 division. :"(